src/ai-dialog-footer.ts

```diff
@@ -27,6 +27,9 @@ export class AiDialogFooter {
   @bindable public buttons: any[] = [];
   @bindable public useDefaultButtons: boolean = false;
 
+  /**
+   * @internal
```

Comment:
Better to simply make `inject` private since it has no bearing on the usage, only on the output API?